src/libpgmoneta/wf_restore.c

```diff
@@ -50,7 +50,6 @@ static int recovery_info_teardown(int, char*, struct node*, struct node**);
 static char* get_user_password(char* username);
 static void create_standby_signal(char* basedir);
 
-
```

Comment:
Skip white space change - so revert

Reply:
I reverted the changes, and then uncrustify just changed it back again.  Which version of uncrustify are you using right now?